src/main/java/com/tenx/ms/retail/product/utils/ProductConverter.java

```diff
@@ -0,0 +1,40 @@
+package com.tenx.ms.retail.product.utils;
+
+import com.tenx.ms.retail.product.domain.ProductEntity;
+import com.tenx.ms.retail.product.rest.dto.Product;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.function.Function;
+
+@Component
+public final class ProductConverter {
+    public static Function<Product, ProductEntity> convertToProductEntity = (Product product) -> {
+        ProductEntity productE = new ProductEntity();
+        StoreEntity storeE = new StoreEntity();
+
+        storeE.setStoreId(product.getStoreId());
+
+        productE.setProductId(product.getProductId());
```

Comment:
Same comments as for the Order Converter, please check for that on all converters